corpus/records.txt

```diff
@@ -209,6 +209,25 @@ record Teacher(string FirstName, string LastName, string Subject) : Person(First
       (identifier)
       (identifier))))
 
+=====================================
+Record inheritance with generic base
+=====================================
+
+record Teacher() : Entity<Person>(), I1;
```

Comment:
I think the simpler, non constructor calling base variant can also be generic:
```
record A : System.IEquatable<A>;
```

Reply:
Oops, good catch. Included in latest commit